docs/Open Data Model/Open Data Model.md

```diff
@@ -0,0 +1,755 @@
+This data type includes information about the network, which can be gleaned from Whois servers, asset databases and other similar data sources.
+
+ - User context data
+This data type includes information from user and identity management systems including Active Directory, Centrify, and other identity and access management systems.
+
+ - Endpoint context data
+This data includes information about endpoint systems (servers, workstations, routers, switches, etc.) and can be sourced from asset management systems, vulnerability scanners, and endpoint  management/detection/response systems such as Webroot, Tanium, Sophos, Endgame, CarbonBlack, Intel Security ePO and others.
+
+ - File context data** (ROADMAP ITEM)**
+This data includes contextual information about files and can be sourced from systems such as FireEye, Application Control and others.
+
+ - Threat intelligence context data **(ROADMAP ITEM)**
+This data includes contextual information about URLs, domains, websites, files and others.
+
+**Naming Convention**
+-----------------
+
+A naming convention is needed for the Open Data Model to represent common attributes across vendor products and technologies. The naming convention is described below.
+
+**Prefixes**
+--------
+
+|  Prefix | Description  |  
+|---|---|
+|  src | Corresponds to the “source” fields within a given event (i.e. source address)|  
+|  dst | Corresponds to the “destination” fields within a given event (i.e. destination address) |  
+|  dvc | Corresponds to the “device” applicable fields within a given event (i.e. device address) and represent where the event originated  |  
+| fwd  | Forwarded from device   |  
+| request | Corresponds to requested values (vs. those returned, i.e. “requested URI”) |  
+| response  | Corresponds to response value (vs. those requested) |  
+| file  |  Corresponds to the “file” fields within a given event (i.e. file type) |  
+| user  | Corresponds to user attributes (i.e. name, id, etc.)  |  
+| xlate  | Corresponds to translated values within a given event (i.e. src_xlate_ip for “translated source ip address” |  
+| in  | Ingress|  
+| out | Egress |  
+| new | New value |  
+| orig | Original value |  
+| app | Corresponds to values associated with application events |  
+
+
+**Security Event Log/Alert Data Model**
+-----------------------------------
+
+The data model for security event logs/alerts is detailed in the below. The attributes are categorized as follows:
+
+ - Common -attributes that are common across many device types
+ - Device -attributes that are applicable to the device that generated the event
+ - File -attributes that are applicable to file objects referenced in the event
+ - Endpoint -attributes that are applicable to the endpoints referenced in the event
+ - User- attributes that are applicable to the user referenced in the event
+ - Proxy - attributes that are applicable to proxy events
+ - Protocol
+
+> DNS - attributes that are specific to DNS events
```

Comment:
I think the > are not needed here. This should be a sublist (see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)